### PR TITLE
Linux CI - Caching of Flatpak runtimes.

### PR DESCRIPTION
**Pull request by VocalFan**
Comparing from the latest Dev Build in obhq's builds VS latest build in this PR (I made Linux run twice so it would use the cache)

No cache: 4:50
Flatpak Runtime cache: 4:07

Do note that since the flatpak runtimes are hard set, it'll only invalidate cache if ci-linux.yml is modified.

### CODE REVIEW

File: .github/workflows/ci-linux.yml [hunk 1]
```diff
@@ -25,5 +32,5 @@ jobs:
     - name: Upload Flatpak
       uses: actions/upload-artifact@v3
       with:
-        name: obliteration-linux
+        name: obliteration-linux64
```
Comment (ultimaweapon) [inline .github/workflows/ci-linux.yml:35]: Now we are going to support both AMD64 and ARM64 so this name is ambiguous. We should rename it to something like `linux-x86_64` for Linux and `windows-x64` for Windows.

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

**Comment by VocalFan** [comment]
I made it be:

Intel/AMD: x64
Arm: arm64

Just to make it short and simple.

File: .github/workflows/ci-linux.yml [hunk 2]
```diff
@@ -32,5 +32,5 @@ jobs:
     - name: Upload Flatpak
       uses: actions/upload-artifact@v3
       with:
-        name: obliteration-linux64
+        name: obliteration-linux-x64
```
Comment (ultimaweapon) [inline .github/workflows/ci-linux.yml:35]: Better to use `x86_64`, `x86-64` or `amd64` because x64 is not an official architecture name on Linux world.

**Review by ultimaweapon — CHANGES_REQUESTED** [review]

File: .github/workflows/ci-linux.yml (hunk 2 above)
Comment (VocalFan) [reply .github/workflows/ci-linux.yml:35]: This is a hate crime.

**Review by ultimaweapon — APPROVED** [review]

**Pull request merged**